### PR TITLE
Edit caption to "Winter is coming" in KristaRutz file

### LINKED ISSUE #4: Add a Caption To an Existing Slide
## Can you think of a better caption?
Each slide can have more than one caption. Check out the current slide show and add a caption (or two).
If you need help, check out the Appendix in the training manual.

### PR DESCRIPTION
**Pull request by KristaRutz**
close #4

### CODE REVIEW

**Review by thesamwong — CHANGES_REQUESTED** [review]
Favourite GoT character?

**Comment by KristaRutz** [comment]
> Favourite GoT character?

I would say Dany but by the end... Sansa tbh

File: _slides/16-KristaRutz.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Winter is coming!
```
Comment (herenhuang) [inline _slides/16-KristaRutz.md:6]: You are riiight!

**Review by herenhuang — APPROVED** [review]
looks good ~

**Review by thesamwong — APPROVED** [review]

**Pull request merged**